code/controllers/subsystem/atoms.dm

```diff
@@ -35,9 +35,9 @@ SUBSYSTEM_DEF(atoms)
 	init_start_time = world.time
 	setupGenetics() //to set the mutations' sequence
 
-	initialized = INITIALIZATION_INNEW_MAPLOAD
+	set_tracked_initalized(INITIALIZATION_INNEW_REGULAR, "base state") // This gets overidden immediately in InitializeAtoms
 	InitializeAtoms()
-	initialized = INITIALIZATION_INNEW_REGULAR
+	base_initialized = INITIALIZATION_INNEW_REGULAR // Backup in case the base state is removed for some reason
 
```

Comment:
this makes me very uncomfortable because this system breaks pretty consistently, why touch it here?